BrewMesModulesParent/common/src/main/java/com/brewmes/common/util/machinenodes/AdminNodes.java

```diff
@@ -1,4 +1,4 @@
-package com.brewmes.subscriber.util;
+package com.brewmes.common.util.machinenodes;
```

Comment:
You should try to avoid making changes like this in a pr that should only contain the implementation of the `MachineServiceImpl`.